### ISSUE TITLE
npx @adobe/reactor-sandbox failing due to "Error: Cannot find module '../../app/constants'"

### ISSUE THREAD

**Issue by roman-bronis** [adobe/reactor-sandbox#60]
### Expected Behaviour
Having sandbox running after `npx @adobe/reactor-sandbox`.

### Actual Behaviour
Sandbox fails to start due to 
```
node:internal/modules/cjs/loader:943
  throw err;
  ^

Error: Cannot find module '../../app/constants'
```
Full output below.

### Reproduce Scenario (including but not limited to)
Starting sandbox.
Replicated on MacOS (node 16.1.0, npm 7.12.0) and Linux (node 16.0.0, npm 7.11.1), same behavior.

#### Steps to Reproduce
npx @adobe/reactor-sandbox

#### Sample Code that illustrates the problem
```bash
[guest@test test]$ npx @adobe/reactor-sandbox
node:internal/modules/cjs/loader:943
  throw err;
  ^

Error: Cannot find module '../../app/constants'
Require stack:
- /home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/tasks/helpers/editorRegistry.js
- /home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/tasks/run.js
- /home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.Module._load (node:internal/modules/cjs/loader:773:27)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/tasks/helpers/editorRegistry.js:15:23)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Function.Module._load (node:internal/modules/cjs/loader:828:14)
    at Module.require (node:internal/modules/cjs/loader:1012:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/tasks/helpers/editorRegistry.js',
    '/home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/tasks/run.js',
    '/home/guest/.npm/_npx/d32d827182dc59bf/node_modules/@adobe/reactor-sandbox/src/cli.js'
  ]
}
```

**Comment by brenthosie** [comment]
@adobe export issue to Jira project PDCL as Bug

**Comment by brenthosie** [comment]
Resolved by v11.1.6

**Comment by roman-bronis** [comment]
@brenthosie Thank you!
